experimental/dds/tree2/src/shared-tree-core/editManagerSummarizer.ts

```diff
@@ -90,19 +84,7 @@ export class EditManagerSummarizer<TChangeset> implements Summarizable {
 		services: IChannelStorageService,
 		parse: SummaryElementParser,
 	): Promise<void> {
-		let schemaBuffer: ArrayBufferLike;
-		if (await services.contains(blobKey)) {
-			const handleBuffer = await services.readBlob(blobKey);
-			const handleString = bufferToString(handleBuffer, "utf-8");
-			const handle = parse(handleString) as IFluidHandle<ArrayBufferLike>;
-			schemaBuffer = await handle.get();
-		} else {
-			assert(
-				await services.contains(stringKey),
-				0x42b /* EditManager data is required in summary */,
-			);
-			schemaBuffer = await services.readBlob(stringKey);
-		}
+		const schemaBuffer: ArrayBufferLike = await services.readBlob(stringKey);
```

Comment:
Do we still want to assert that the blob exists? If it doesn't, then we're gonna fail down below in some weird way right. But, I suppose we can safely assume that the blob is always there given that we know the write format. So, either way :)

Reply:
`readBlob` throws on failure rather than returning undefined.